proxy/fyde_proxy_bm_vm.md

```diff
@@ -1,60 +1,82 @@
 # Install Fyde Proxy in Bare Metal / Virtual Machine
 
 - Supported distributions:
+  - CentOS 7
 
-  - [CentOS 7](##centos-7)
+- Instalation requires a valid [Proxy Enrollment Link](../console/configurations/add_proxy.md#adding-a-proxy)
 
-## Centos 7
+- Choose [**Install script**](##install-script) or [**Manual steps**](##manual-steps) to proceed
+
+## Install script
+
+- Please note that the steps below will execute scripts obtained externally
+- We advise to inspect the content before execution
+
+1. Download and execute script
+
+    ```sh
+    curl -fsSL https://url.fyde.me/install-fyde-proxy-centos \
```

Comment:
@GMartinez-Sisti how about `curl -fsSL https://url.fyde.me/install-fyde-proxy-centos | sudo bash`

Reply:
That works, but I'm assuming the sysadmin will first want to take a look at the script before executing. WDYT?

Reply:
Change it according to suggestion. Please approve again.

Reply:
I'd say that if they want (and they should), they'll know what to do :)